lib/internal/Magento/Framework/Mail/Message.php

```diff
@@ -189,4 +189,12 @@ public function setBodyText($text)
         $this->setMessageType(self::TYPE_TEXT);
         return $this->setBody($text);
     }
+
+    /**
+     * @return \Zend\Mail\Message
+     */
+    public function getZendMessage(): \Zend\Mail\Message
```

Comment:
We cannot add public method in the implementation without adding it to the interface. But the interface is marked as `@api` and this is [backward incompatible change](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/). Moreover our intention was not to expose Zend objects at all. So the best approach is to create our wrapper for Zend\Message\Headers class, create a new interface like `MessageHeadersInterface` implementing 'MailMessageInterface` and add headers setter to this class implementing the new interface.

Reply:
Thanks for the feedback, will do this when I find some time.